src/cli/workspace-publish.ts

```diff
@@ -0,0 +1,60 @@
+/**
+ * @license
+ * Copyright (c) 2017 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import * as chalk from 'chalk';
+import * as inquirer from 'inquirer';
+import {Workspace, WorkspaceRepo} from 'polymer-workspaces';
+
+export default async function run(
+    workspace: Workspace, reposToConvert: WorkspaceRepo[]) {
+
+  console.log(
+      chalk.dim('[1/3] ') + chalk.magenta(`Setting up publish to npm...`));
+
+  const whoAmI = await reposToConvert[0].npm.whoami();
+  console.log(`npm whoami: ${whoAmI}`);
+  const {publishTag} = (await inquirer.prompt([
```

Comment:
nit: why is this in parens `(await ...)`?

Reply:
Good catch, removing...